projects/vdk-control-cli/src/vdk/internal/control/configuration/vdk_config.py

```diff
@@ -208,7 +209,25 @@ def read_configuration(self, section, option, fallback=None):
             log.debug(f"Reading configuration file: {configuration_file_path} ...")
 
             config_parser = ConfigParser()
-            config_parser.read(configuration_file_path)
+            try:
+                config_parser.read(configuration_file_path)
+            except (MissingSectionHeaderError, Exception) as e:
+                log.debug(e, exc_info=log.getEffectiveLevel() == logging.DEBUG)
+                raise VDKException(
+                    what="Cannot parse the Data Job configuration file.",
+                    why=f"Configuration file config.ini is probably "
+                    f"corrupted. Error: {e}",
+                    consequence="Cannot deploy and configure the data job "
+                    "without "
+                    " properly set config.ini file.",
+                    countermeasure="config.ini must be UTF-8 compliant. "
+                    "Make sure the file does not contain "
+                    "special "
+                    "Unicode characters, or that your text "
+                    "editor "
+                    "has not added such characters somewhere "
+                    "in the file.",
```

Comment:
Did the precommit hooks mess up the text formatting here?

Reply:
Either the hooks, or my IDE :D
Fixed, I think. Thanks